doorway-ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -88,8 +89,7 @@ const ListingCard = (props: ListingCardProps) => {
     contentProps,
     tableProps,
   } = props
-  const { LinkComponent } = useContext(NavigationContext)
-  const linkRef = React.useRef<HTMLAnchorElement>(null)
+  const linkRef = useRef<HTMLAnchorElement>(null)
```

Comment:
Fix a bug where the full listing card wasn't actually clickable to the listing, even though we were semi-indicating it was with the pointer cursor